libraries/chain/asset_evaluator.cpp

```diff
@@ -407,20 +407,31 @@ void_result asset_update_bitasset_evaluator::do_evaluate(const asset_update_bita
  * @param db the database
  * @param bdo the actual database object
  * @param asset_to_update the asset_object related to this bitasset_data_object
+ * @returns true if the feed price is changed
```

Comment:
Can change to return true only if (feed changed) AND (after_hf_890)

Reply:
Done.